docs/platforms/python/integrations/bottle/index.mdx

```diff
@@ -73,22 +73,22 @@ sentry_sdk.init(
 
 ```
 
-You can pass the following keyword arguments to `BottleIntegration()`:
+Each of the following options are supported as keyword arguments to `BottleIntegration()`.
```

Comment:
It says 'Each of the following options' here, but the only option I see listed below that is `transaction_style`, is that correct? If that's the case, I think this statement can be omitted.

Reply:
#11515 (which I will merge directly after this PR as it is based on this one) will add an additional option. So, this can stay, as it will soon be correct.